src/ingest.py

```diff
@@ -220,6 +234,13 @@ def main():
 
                 logger.info("Ingesting %s", resources)
                 refs = None
+                if is_lfa:
+                    resources_photodiode = [
```

Comment:
The Internet says you could write this as:
```
resources_photodiode = []
resources = []
for resource in resources:
    (resources_photodiode if re.search(r"MTCamera/photodiode.*_photodiode.ecsv$", resource) else resources).append(resource)
```
Even the fully explicit:
```
for resource in resources:
    if re.search():
        resources_photodiode.append(resource)
    else:
        resources.append(resource)
```
might be clearer.

Reply:
Rewritten to more like the second example